vowpalwabbit/example.cc

```diff
@@ -14,10 +14,8 @@ float calculate_total_sum_features_squared(bool permutations, example& ec)
   float sum_features_squared = 0.f;
```

Comment:
A micro benchmark for this function would be good so that most of the effects for feature_squared calculation can be evaluated in isolation.  And this branch can be compared to master.

Reply:
I don't know what the benchmark would evaluate? The runtime of this function is dependent on the interactions and extent interactions. The latter does not exist in master and the former is an equivalent operation.

Reply:
Does not have to be in this PR.    Trying to anticipate questions around perf impact.

Trying to answer the following questions with benchmarks:  1) How much does new style interaction cost relative to old style interactions... for 2, 3, 4 etc interactions.  2) This function should be affected by change in interaction functionality.  Is it quantifiable in old vs new?

Reply:
I can whip up a couple of benchmarks comparing equivalent interaction using each system.